File: Vari/8/Solution.cpp
#include <string>

class Solution {
public:
    bool isPalindrome(int x) {
        string strInt = to_string(x);

        for(int i = 0; i < strInt.length()/2; i++)
            if(strInt[i] != strInt[strInt.length()-1-i])
                return false;

        return true;
    }
};

File: Vari/8/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
        String stringInteger = Integer.toString(x);

        for(int i = 0; i < stringInteger.length()/2; i++)
            if(stringInteger.charAt(i) != stringInteger.charAt(stringInteger.length()-1-i))
                return false;

        return true;
    }
}